src/lib.rs

```diff
@@ -94,13 +94,16 @@ impl UPClientZenoh {
     /// ```
     /// #[tokio::main]
     /// # async fn main() {
-    /// use up_transport_zenoh::{Config, UPClientZenoh};
-    /// let upclient = UPClientZenoh::new(Config::default(), String::from("MyAuthName"))
+    /// use up_transport_zenoh::{UPClientZenoh, ZenohConfig};
+    /// let upclient = UPClientZenoh::new(ZenohConfig::Config::default(), String::from("MyAuthName"))
```

Comment:
For example, if this were `zenoh_config::Config::default()` here this would read more naturally to me in Rust.